proof_test.go

```diff
@@ -942,3 +981,56 @@ func TestProofOfAbsenceBorderCase(t *testing.T) {
 		t.Fatal("differing commitment for child #0")
 	}
 }
+
+func TestGenerateProofWithOnlyAbsentKeys(t *testing.T) {
```

Comment:
this is taken from 400, should not be merged in any case as the version over there is better